src/Elcodi/Component/User/Entity/Abstracts/AbstractUser.php

```diff
@@ -38,6 +38,21 @@
         DateTimeTrait,
         EnabledTrait;
 
+    const GENDER_UNDEFINED = 0;
```

Comment:
The undefined value is used for two use cases:
1. https://github.com/elcodi/elcodi/issues/999#issuecomment-160705453 (last point)
2. if a user doesn't want to define their sex, don't know the reason exactly, but I know it's a common thing around the net